src/mmw/js/src/draw/templates/drawTool.html

```diff
@@ -13,18 +13,24 @@
         <ul class="dropdown-menu draw-pane-list">
             {% for item in toolData.items %}
                 <li class="draw-pane-list-item" role="presentation">
+                    <button class="draw-pane-list-item-button" role="button" tabindex="-1"
```

Comment:
Except for the disable styling the button should look identical as to when it was a `<span>`